dist/amd/aurelia-binding.js

```diff
@@ -47,7 +47,7 @@ define(['exports', 'aurelia-logging', 'aurelia-pal', 'aurelia-task-queue', 'aure
   var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
     return typeof obj;
   } : function (obj) {
-    return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj;
```

Comment:
Done. Sorry, this is my first PR for Aurelia. I misunderstood the line in the [Contributor's Guide](https://github.com/DurandalProject/about/blob/master/CONTRIBUTING.md)

> Build your changes locally to ensure all the tests pass

to mean commit up the built files.